app/utils/db.py

```diff
@@ -503,23 +504,34 @@ def find_and_update(collection, query, document, operation="$set"):
     return ret_val
 
 
-def delete(collection, spec_or_id):
-    """Remove a document or multiple documents from the collection.
+def delete_by_id(collection, id_):
```

Comment:
Ditto - maybe `oid` would be better than `id_`.

Reply:
Corrected.